doc/source/math/ref_frame.rst

```diff
@@ -19,14 +19,14 @@ documentation:
     system.  kOS inherits this behavior from KSP.
 
 In all the reference frames mentioned below, the orientation of 
-the axes is **left-handed**.  This means that if you imagine opening your
+the axes is **left-handed**. This means that if you imagine opening your
```

Comment:
I also learned double-spacing in school and fought for it valiantly until a coworker finally convinced me with [this info][gg], or a version of that story. Realizing that almost all modern writing uses single spaces, I made myself switch. It wasn't nearly as hard as I thought.
Then I found [the rebuttal][r], and now I care both more and less. I use the single space by modern convention, but I've stopped evangelizing either way.

In any case, I don't even remember changing that intentionally and it makes no difference in source. Reverted.

[gg]: http://www.quickanddirtytips.com/education/grammar/two-spaces-after-a-period
[r]: http://www.heracliteanriver.com/?p=324